src/pocketmine/command/CommandReader.php

```diff
@@ -34,21 +35,36 @@ public function __construct(){
 		$opts = getopt("", ["disable-readline"]);
 		$this->readline = (extension_loaded("readline") and !isset($opts["disable-readline"]));
 		$this->start();
+		echo(\pocketmine\PATH);
+		$this->getClassLoader()->addPath(\pocketmine\PATH . "src");
```

Comment:
I think the class loader already has this path. I think only register() is necessary.